### DISCUSSION TITLE
My Old Account has been Suspended

### DISCUSSION THREAD

**Discussion by GadagiPooja** [community/community#140655] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Subject: Request for Account Reinstatement - Urgent

Dear GitHub Support,

I am writing to express my deep distress regarding the recent suspension of my old GitHub account (username: PoojaGadagi1612). I was notified that my account was suspended due to a violation, but I genuinely believe I have not engaged in any activity that would warrant such an action.

This account holds all my projects, which I have worked on tirelessly over the years. I am currently in the midst of placement season, and losing access to my account has jeopardized my ability to showcase my achievements and projects. I am devastated, as all my hard work has disappeared in an instant.

Prior to the suspension, I noticed that a new repository had been added to my account, which was not mine. I suspect that this unauthorized addition may be the reason for the suspension. I had no involvement with this repository, and it is heartbreaking to think that this might have led to the loss of my account.

I have now created a new GitHub account, because unfortunately, I am unable to access GitHub Support from my old account due to the lack of authorization. I am desperately pleading with you to review my case and reinstate my original account. This is an urgent request as my career prospects are at stake.

Please help me recover my account and all the work that I’ve poured my heart and soul into.

Sincerely,
Pooja Gadagi
(Old Username: PoojaGadagi1612)

![Screenshot 2024-10-05 100034](https://github.com/user-attachments/assets/17b1d632-0b38-4a47-9e8e-796f2f4668f5)

**Comment by ghost** [comment]
even my acc was suspended a few days ago randomly, did u get any update? @GadagiPooja

**Reply by GadagiPooja** [reply]
Not Yet

**Comment by Topromm** [comment]
If you haven't yet, make sure to reach out to GitHub through a [ticket](https://support.github.com/request) explaining your current situation. 

Going forward, it would be a good idea to back up your important repos either locally or in the cloud. You could also mirror your repos on GitLab or Bitbucket.

**Reply by GadagiPooja** [reply]
Yes , I'll do that 
Thank you!

**Comment by ebndev** [comment, marked as answer]
Hi @GadagiPooja,

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
